pkg/kwokctl/runtime/util.go

```diff
@@ -93,3 +99,35 @@ func GetLogVolumes(ctx context.Context) ([]internalversion.Volume, error) {
 
 	return volumes, nil
 }
+
+type dirMountSet struct {
+	mounts map[string]struct{}
+	err    error
+}
+
+func (m *dirMountSet) add(logsFile string) {
+	if m.err != nil {
+		return
+	}
+	if m.mounts == nil {
+		m.mounts = make(map[string]struct{})
+	}
+	abs, err := path.Expand(logsFile)
+	if err != nil {
+		m.err = err
```

Comment:
This error does not seem to have been handled.

Reply:
Oh good catch, I forgot to handle that part